app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/edit/nutrition/ProductEditNutritionFactsFragment.kt

```diff
@@ -551,10 +555,11 @@ class ProductEditNutritionFactsFragment : ProductEditFragment() {
         // Add no nutrition data entry to map
         if (binding.checkboxNoNutritionData.isChecked) {
             targetMap[ApiFields.Keys.NO_NUTRITION_DATA] = "on"
```

Comment:
So we send the boolean to deactivate, but do we send the boolean to reactivate, along or before nutrients ?

Reply:
You're right, this is not the case right now.

Currently in the code, the app checks if the field is null (= no nutrition data) or == "ON"
So I will just send a null in the other cases. Otherwise I am afraid of generating some bugs

Reply:
That should be OK right now, with the new commit #9ce02dd